complete/build.gradle

```diff
@@ -85,15 +76,6 @@ bootRun {
 
 }
 
-webdriverBinaries {
-    chromedriver {
-        version = '77.0.3865.40'
-    }
-    geckodriver {
-        version = '0.24.0'
-    }
-}
-
 tasks.withType(Test) {
     systemProperty "geb.env", System.getProperty('geb.env')
```

Comment:
if we are not using geb, we can remove the geb system properties configurations.